### PR TITLE
5.7.36 psmerge part 2

### PR DESCRIPTION
**Pull request by nitendra-bhosle**

### CODE REVIEW

**Comment by nitendra-bhosle** [comment]
Jenkins run : https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-param/252/

**Comment by venkatesh-prasad-v** [comment]
Looks good. Can you please check if the failure of `main.mysqldump_gtid_state` on CentOS-7 build is related to this merge?https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-param/252/CMAKE_BUILD_TYPE=RelWithDebInfo,DOCKER_OS=centos%3A7/testReport/

**Review by kamil-holubicki — COMMENTED** [review]
Looks good. No other question than already asked about `mysqldump_gtid_state`

**Comment by nitendra-bhosle** [comment]
New Jenkins : https://pxc.cd.percona.com/view/PXC%205.7/job/pxc-5.7-param/258/

**Review by kamil-holubicki — APPROVED** [review]
lgtm

File: mysql-test/t/mysqldump_gtid_state.test [hunk 1]
```diff
@@ -1,176 +0,0 @@
---echo #
```
Comment (venkatesh-prasad-v) [inline mysql-test/t/mysqldump_gtid_state.test]: Did you investigate why this test is failing?

**Review by venkatesh-prasad-v — APPROVED** [review]
LGTM

**Pull request merged**